pkg/api/azenvtypes.go

```diff
@@ -100,8 +100,8 @@ var (
 		DockerComposeDownloadURL: "https://github.com/docker/compose/releases/download",
 	}
 
-	//DefaultUbuntuImageConfig is the default Linux distribution.
-	DefaultUbuntuImageConfig = AzureOSImageConfig{
+	//Ubuntu1604ImageConfig is the default Linux distribution.
+	Ubuntu1604ImageConfig = AzureOSImageConfig{
```

Comment:
just a few renames for consistency, some image configs had the word "default" in their name, others didn't.